### DISCUSSION TITLE
LoanableCollection in Python fastdds

### DISCUSSION THREAD

**Discussion by swarm59** [eProsima/Fast-DDS#2674] [Q&A]
I guess if LoanableCollection and its specializations, i.e. LoanableArray, and LoanableSequence are available in the Python fastdds, for the Ubuntu Focal platform.
If so, does exist any example ?

Thanks for the support,
Francesco

**Comment by JLBuenoLopez** [comment]
Hi @swarm59,

Fast DDS Python bindings is a really recent project so the only example available right now is the [`HelloWorldExample`](https://github.com/eProsima/Fast-DDS-python/tree/main/fastdds_python_examples). The idea is to complete and add some more examples, but it is not in our roadmap in the short term. Maybe you can look into the [tests](https://github.com/eProsima/Fast-DDS-python/tree/main/fastdds_python/test/api) where the supported API is checked.

**Reply by swarm59** [reply]
Hi @JLBuenoLopez-eProsima ,
thanks for the clarification.
Sorry but I still did not understand whether LoanableCollection and its specializations, i.e. LoanableArray, and LoanableSequence are available in the Python fastdds.
My best,
Francesco

**Comment by JLBuenoLopez** [comment]
@swarm59,

Please refer to the [Python API reference](https://fast-dds.docs.eprosima.com/en/latest/fastdds/python_api_reference/python_api_reference.html) to know if it is available in Fast DDS Python Bindings. [LoanableCollection](https://fast-dds.docs.eprosima.com/en/latest/fastdds/python_api_reference/dds_pim/core/loanablecollection.html#loanablecollection) is documented there, so probably the other two are also supported (even though the doxygen documentation generation seem to be failing). However, there are included into the API reference so they should be supported.

**Reply by swarm59** [reply]
Many Thanks,
